final-project/styles/main.scss

```diff
@@ -2,12 +2,826 @@
+}
+
+@mixin tablet-breakpoint {
+  @media only screen and (min-width: rem-calc(768px)) {
+    @content;
+  }
+}
+
+@mixin desktop-breakpoint {
+  @media only screen and (min-width: rem-calc(992px)) {
+    @content;
+  }
+}
 
 * {
   padding: 0;
   margin: 0;
+  padding: 0;
+  margin: 0;
+  font-family: Quicksand;
+  font-variant:100;
 }
 
+button { border:none; } 
+
 h1 {
   color: $primary-color;
+  font-weight: 200;
+
+}
+
+header{
+  display:flex;
+  flex-direction: column;
+  h1{
+    text-align: center;
+    font-size: rem-calc(48px);
+    font-weight: 300;
+    color: $header__text-color;
+  }
+  h2{
+    text-align: center;
+    font-size: rem-calc(36px);
+    font-weight: 500;
+    color: $header__text-color;
+  }
+  h3{
+    text-align: center;
+    font-size: rem-calc(28px);
+    font-weight:500;
+    color: $header__text-color;
+  }
+  span{
+    font-family: Roboto;
+    font-weight:500;
+    color: $header__text-color;
+    font-size: rem-calc(10px);
+    text-transform: uppercase;
+    text-align: center;
+  }
+}
+
+p{
+  font-family: Roboto;
+  font-weight:400;
+  color:#8D96BD;
+  font-size: rem-calc(15px);
+  text-align: center;
+}
+
+.blog{
+  display: flex;
+  flex-direction: column;
+  background-color: $alter-section-color;
+  padding: 0 rem-calc(14px);
+  &__content{
+    display: flex;
+    left: 0;
+    position: relative;
+    transition: left 3s;
+    &__article{
+      width: 100%;
+      max-width: rem-calc(992px);
+      display: flex;
+      flex-direction: column;
+      border: 1px solid #D8E3FE;
+      background-color: white;
+      &__image{
+        display: flex;
+        background-color: $alter-section-color;
+        margin: rem-calc(16px);
+      }
+      &__info{
+        display: flex;
+        flex-direction: column;
+        margin: rem-calc(16px) rem-calc(16px);
+        @include desktop-breakpoint {
+          margin: rem-calc(40px) rem-calc(16px) rem-calc(40px) 0 ;
+        }
+      }
+      &__actions{
+        &__read{
+          background-color: $action-button-color;
+          width: 100%;
+          height: rem-calc(34px);
+          color:#3A79FF;
+          @include tablet-breakpoint {
+            width: rem-calc(178px);
+          }
+          @include desktop-breakpoint {
+            width: rem-calc(109px);
+          }
+        }
+        &__bookmark{
+          display: none;
+          @include tablet-breakpoint {
+            display: inline;
+          }
+          @include desktop-breakpoint {
+            display: inline;
+          }
+        }
+      }
+      @include desktop-breakpoint {
+        width: rem-calc(992px);
+        flex-direction: row;
+      }
+    }
+  }
+  &__visor{
+    display: flex;
+    width: 100%;
+    max-width: rem-calc(304px);
+    margin: 0 auto;
+    overflow: hidden;
+    @include tablet-breakpoint {
+      max-width: rem-calc(677px);
+    }
+    @include desktop-breakpoint {
+      max-width: rem-calc(994px);
+    }
+  }
+  span{
```

Comment:
Is ok to have this kind of global declarations but I think they can be hard to find if they're not together at the top of your document, also you can have cascading issues.